terraform/deployments/cluster-services/argo.tf

```diff
@@ -76,76 +78,45 @@ resource "helm_release" "argo_notifications" {
   repository = "https://argoproj.github.io/argo-helm"
   version    = "1.5.1" # TODO: Dependabot or equivalent so this doesn't get neglected.
   values = [yamlencode({
-    argocdUrl = "https://${local.argo_host}"
+    # Configured in argo-services Helm chart
+    "cm.create" = false
```

Comment:
Thanks, I've changed this. Interestingly this _seemed_ to work when using `helm_release`, though I'm probably misremembering, since it would be strange if this resource used `set` (e.g. `--set=cm.create=false`) behind the scenes (especially given there's a `set {}` block on the resource!).